CLAUDE.md

```
@@ -0,0 +1,105 @@
+# solid-connection-infra
+
+solid-connection 서비스의 AWS 인프라를 Terraform으로 관리하는 IaC 레포지토리입니다.
+
+## 디렉토리 구조
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)